packages/compass-import-export/src/utils/get-user-data-file-path.ts

```diff
@@ -0,0 +1,9 @@
+import path from 'path';
+import { getStoragePaths } from '@mongodb-js/compass-utils';
+const { appName, basepath } = getStoragePaths() || {};
+
+export function getUserDataFolderPath() {
+  if (appName === undefined || basepath === undefined) return;
```

Comment:
This should maybe assert or just throw an error in this case because you now have a mixed return type.

Reply:
We throw now 👍